packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/iOS/signin.md

```diff
@@ -1,20 +1,4 @@
-Before using the SDK, you have to create a new `OktaOidc` object. You can instantiate `OktaOidc` without parameters. If you do that, the SDK uses Okta.plist for configuration values. Alternatively, you can create the `OktaOidc` instance with a custom configuration.
-
-```swfit
-import OktaOidc
-
-// Use the default Okta.plist configuration
-let oktaOidc = OktaOidc()
-
-// Use configuration from another resource
-let config = OktaOidcConfig(/* plist */)
-// or: let config = OktaOidcConfig(/* dictionary */)
-
-// Instantiate OktaOidc with custom configuration object
-let oktaOidc = OktaOidc(configuration: config)
-```
-
-Then, you can start the authorization flow by simply calling `signInWithBrowser`. When successfully authorized, this operation returns a valid `OktaOidcStateManager` instance in its callback:
+Start the authorization flow by simply calling `signInWithBrowser` on `OktaOidc`. After a successful sign-in, this operation returns a valid `OktaOidcStateManager` instance in its callback:
```

Comment:
Start the authorization flow by simply calling `signInWithBrowser` on `OktaOidc`. After a successful sign in, this operation returns a valid `OktaOidcStateManager` instance in its callback: